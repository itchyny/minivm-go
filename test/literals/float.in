print 3.14
print 6.12
print 123.45
