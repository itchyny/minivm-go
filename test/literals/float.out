3.14
6.12
123.45
